doc/changes/latest.inc

```diff
@@ -72,6 +72,8 @@ Changelog
 Bug
 ~~~
 
+- Fix a bug in :meth:`mne.source_estimate.MixedSourceEstimate.plot_surface` by `Christian O'Reilly`_.
```

Comment:
This will not link properly (and CircleCI will fail) because `MixedSourceEstimate` is documented in the `mne` namespace

https://mne.tools/dev/generated/mne.MixedSourceEstimate.html#mne.MixedSourceEstimate

```suggestion
- Fix a bug in :meth:`mne.MixedSourceEstimate.plot_surface` that prevented plotting in latest PySurfer by `Christian O'Reilly`_.
```